New Broken Links (#2397)
MicrosoftDocs/edge-developer: microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/console-integration.md

![The DevTools Console inside the extension as an own tab](./console-integration-images/console-full.png)

You can:

> *  View [log messages](/microsoft-edge/devtools-guide-chromium/console/console-log).

Reviewer: ```suggestion
*  View [log messages](../../devtools-guide-chromium/console/console-log.md).
```
try to test this link in VS Code b/c ../ levels

live:
https://learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/console/console-log

> *  Access the `window` object and use the [DOM interaction convenience methods](/microsoft-edge/devtools-guide-chromium/console/console-dom-interaction).

Reviewer: ```suggestion
*  Access the `window` object and use the [DOM interaction convenience methods](../../devtools-guide-chromium/console/console-dom-interaction.md).
```

> *  [Filter the Console](../../devtools-guide-chromium/console/console-filters) and set [live expressions](/microsoft-edge/devtools-guide-chromium/live-expressions.md).

Reviewer: ```suggestion
*  [Filter the Console](../../devtools-guide-chromium/console/console-filters.md) and set [live expressions](../../devtools-guide-chromium/live-expressions.md).
```

> *  [Filter the Console](/microsoft-edge/devtools-guide-chromium/console/console-filters) and set [live expressions](/microsoft-edge/devtools-guide-chromium/live-expressions).

Reviewer: ```suggestion
*  [Filter the Console](../../devtools-guide-chromium/console/console-filters) and set [live expressions](/microsoft-edge/devtools-guide-chromium/live-expressions.md).
```
